src/audio/src/src/src_small_int32_table.h

```diff
@@ -8,27 +8,27 @@
 #define __SOF_AUDIO_COEFFICIENTS_SRC_SRC_SMALL_INT32_TABLE_H__
 
 /* SRC conversions */
-#include <sof/audio/coefficients/src/src_small_int32_1_2_2268_5000.h>
-#include <sof/audio/coefficients/src/src_small_int32_1_2_4535_5000.h>
-#include <sof/audio/coefficients/src/src_small_int32_1_3_2268_5000.h>
-#include <sof/audio/coefficients/src/src_small_int32_1_3_4535_5000.h>
-#include <sof/audio/coefficients/src/src_small_int32_2_1_2268_5000.h>
-#include <sof/audio/coefficients/src/src_small_int32_2_1_4535_5000.h>
-#include <sof/audio/coefficients/src/src_small_int32_2_3_4535_5000.h>
-#include <sof/audio/coefficients/src/src_small_int32_3_1_2268_5000.h>
-#include <sof/audio/coefficients/src/src_small_int32_3_1_4535_5000.h>
-#include <sof/audio/coefficients/src/src_small_int32_3_2_4535_5000.h>
-#include <sof/audio/coefficients/src/src_small_int32_3_4_4535_5000.h>
-#include <sof/audio/coefficients/src/src_small_int32_4_3_4535_5000.h>
-#include <sof/audio/coefficients/src/src_small_int32_4_5_4535_5000.h>
-#include <sof/audio/coefficients/src/src_small_int32_5_4_4535_5000.h>
-#include <sof/audio/coefficients/src/src_small_int32_5_6_4354_5000.h>
-#include <sof/audio/coefficients/src/src_small_int32_6_5_4354_5000.h>
-#include <sof/audio/coefficients/src/src_small_int32_7_8_4535_5000.h>
-#include <sof/audio/coefficients/src/src_small_int32_8_7_4535_5000.h>
-#include <sof/audio/coefficients/src/src_small_int32_20_21_4167_5000.h>
-#include <sof/audio/coefficients/src/src_small_int32_21_20_4167_5000.h>
-#include <sof/audio/src/src.h>
+#include "src_small_int32_1_2_2268_5000.h"
+#include "src_small_int32_1_2_4535_5000.h"
+#include "src_small_int32_1_3_2268_5000.h"
+#include "src_small_int32_1_3_4535_5000.h"
+#include "src_small_int32_2_1_2268_5000.h"
+#include "src_small_int32_2_1_4535_5000.h"
+#include "src_small_int32_2_3_4535_5000.h"
+#include "src_small_int32_3_1_2268_5000.h"
+#include "src_small_int32_3_1_4535_5000.h"
+#include "src_small_int32_3_2_4535_5000.h"
+#include "src_small_int32_3_4_4535_5000.h"
+#include "src_small_int32_4_3_4535_5000.h"
+#include "src_small_int32_4_5_4535_5000.h"
+#include "src_small_int32_5_4_4535_5000.h"
+#include "src_small_int32_5_6_4354_5000.h"
+#include "src_small_int32_6_5_4354_5000.h"
+#include "src_small_int32_7_8_4535_5000.h"
+#include "src_small_int32_8_7_4535_5000.h"
+#include "src_small_int32_20_21_4167_5000.h"
+#include "src_small_int32_21_20_4167_5000.h"
+#include "../src.h"
```

Comment:
And this change here https://github.com/thesofproject/sof/blob/2c6c1b4d4fb636ea78dfe1136737c4b6baf5b21c/tools/tune/src/src_export_table_2s.m#L94